src/mixins/filing-template-mixin.ts

```diff
@@ -590,6 +590,8 @@ export default class FilingTemplateMixin extends Mixins(DateMixin) {
         }
       }) || []
     )
+    // TODO: REMOVE MOCK DATA
+    businessSnapshot.shareStructure.resolutionDates = ['2012/04/06', '2014/06/16', '2015/07/05']
```

Comment:
to remove?

Reply:
Definitely! Thank you, will do now and just re apply locally 👍

Reply:
removed